Makefile

```diff
@@ -6,3 +6,6 @@ report:
 
 release:
 	TIMBRE_LEVEL=:warn npx shadow-cljs release main
+
+server:
+	clj -A:dev:datomic -M -m com.example.components.server
```

Comment:
You should not need to include the :dev alias, since it is meant for deps that are only needed in dev.